vmdk_plugin/vmdkops/esx_vmdkcmd.go

```diff
@@ -93,6 +93,9 @@ func (vmdkCmd EsxVmdkCmd) Run(cmd string, name string, opts map[string]string) (
 		msg := fmt.Sprintf("'%s' failed: %v (errno=%d).", cmd, err, int(errno))
 		if errno == syscall.ECONNRESET {
 			msg += " Check that ESX service is running."
```

Comment:
The vmci module can be unloaded between docker commands (in theory)

On Thursday, June 9, 2016, Prashant Dhamdhere <notifications@github.com>
wrote:

> In vmdk_plugin/vmdkops/esx_vmdkcmd.go
> <https://github.com/vmware/docker-volume-vsphere/pull/452#discussion_r66568846>
> :
>
> > @@ -93,6 +93,9 @@ func (vmdkCmd EsxVmdkCmd) Run(cmd string, name string, opts map[string]string) (
> >  		msg := fmt.Sprintf("'%s' failed: %v (errno=%d).", cmd, err, int(errno))
> >  		if errno == syscall.ECONNRESET {
> >  			msg += " Check that ESX service is running."
>
> Rather than embedding these error messages here, how about introducing
> separate init-handshake op. Tools/VMCI/Port error messages doesnt make
> sense after first command is completely successfully.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/vmware/docker-volume-vsphere/pull/452/files/e0fa79b2f90beb9168756ac80a1f92f87e5e7346#r66568846>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAvRNy33R1noDHqoW1B8qj3_ykeGjgJUks5qKQpWgaJpZM4Iyq8D>
> .
>